packages/calypso-products/src/plans-list.tsx

```diff
@@ -2134,6 +2144,26 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getPathSlug: () => 'ecommerce-2-years',
 	},
 
+	[ PLAN_WOOEXPRESS_MEDIUM_MONTHLY ]: {
+		...getPlanWooExpressMediumDetails(),
+		...getMonthlyTimeframe(),
+		availableFor: ( plan ) => [ PLAN_FREE, PLAN_ECOMMERCE_TRIAL_MONTHLY ].includes( plan ),
+		getProductId: () => 1053,
+		getStoreSlug: () => PLAN_WOOEXPRESS_MEDIUM_MONTHLY,
+		getPathSlug: () => 'wooexpress-medium-monthly',
```

Comment:
Assuming this is referring to the `store_product.product_slug`, I think it needs to be this:

```suggestion
		getPathSlug: () => 'wooexpress-medium-bundle-monthly',
```

Reply:
This value is based on a slug that will be accepted in URLs of the form `/checkout/:siteSlug/:pathSlug`, where `pathSlug` can be one of two values:
1. The product slug from `store_products.product_slug`, which would be `wooexpress-medium-bundle-monthly`
2. The `path_slug` values stores in `store_products.extra`, which would be `wooexpress-medium-monthly`

I think it will generally be better to use the path slug, as it is a bit more human-readable.

Reply:
Got, it. Thanks for clarifying.